src/ape_init/_cli.py

```diff
@@ -1,11 +1,47 @@
 import shutil
 from pathlib import Path
+from typing import List
 
 import click
 
 from ape.cli import ape_cli_context
 from ape.utils import github_client
 
+try:
+    import tomllib  # type: ignore[import]
+# backwards compatibility
+except ModuleNotFoundError:
+    import tomli as tomllib
+
+
+def read_dependencies_from_toml(file_path, cli_ctx) -> List[str]:
+
+    with open("./pyproject.toml", "rb") as file:
```

Comment:
this should all happen in the config manage, not in ape-init plugin.
And then you can use the project_manager.path as the base here instead of cwd